### PR TITLE
Upgrading to Selenium 3.3.1

### PR DESCRIPTION
**Pull request by diemol**

### CODE REVIEW

File: scripts/zalenium.sh [hunk 1]
```diff
@@ -206,7 +206,7 @@ trap DockerTerminate SIGTERM SIGINT SIGKILL
 StartUp()
 {
     EnsureDockerWorks
-    CONTAINER_ID=$(cat /proc/self/cgroup | grep -Po '(?<=docker/)([a-z0-9]+)' | head -1)
+    CONTAINER_ID=$(grep -Po '(?<=docker/)([a-z0-9]+)' /proc/self/cgroup | head -1)
```
Comment (diemol) [inline scripts/zalenium.sh]: @a1exmac sometimes 🐱  are useless.
👍

**Comment by scherniavsky** [comment]
:+1:

**Comment by diemol** [comment]
👍

**Comment by elgalu** [comment]
:+1:

**Pull request merged**